### PR TITLE
customscript: update docs/manifest with new version

### PR DESCRIPTION
**Pull request by ahmetalpbalkan**
cc: @boumenot

### CODE REVIEW

File: CustomScript/README.md [hunk 1]
```diff
@@ -2,6 +2,17 @@
 
 Allow the owner of the Azure Virtual Machines to run customized scripts in the VM.
 
+
+# :warning: New Version Notice :warning:
+
+A new version of **Custom Script Extension** is available at https://github.com/Azure/custom-script-extension-linux.
```
Comment (boumenot) [inline CustomScript/README.md]: I would add reasoning as to why you would want to.

**Comment by boumenot** [comment]
LGTM.

**Comment by ahmetalpbalkan** [comment]
clarified. merging

**Comment by ahmetalpbalkan** [comment]
@boumenot can we release a hotfix just so the Description on the API gets updated?

**Pull request merged**